.prettierrc.json

```diff
@@ -27,6 +27,16 @@
       "excludeFiles": [
         "specification/**/*.json"
       ]
+    },
+    {
+      "files": [
+        "**/*.ts"
+      ],
+      "options": {
+        "printWidth": 80,
+        "endOfLine": "auto",
+        "tabWidth": 4
```

Comment:
I recommend copying the settings from the typespec repo:

https://github.com/microsoft/typespec/blob/799cb6af21b5274543701096af8ecb9d5c9bb793/.prettierrc.json#L2-L8